modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/AppendProcessorTests.java

```diff
@@ -179,7 +179,7 @@ public void testAppendingUniqueValueToScalar() throws Exception {
         String field = RandomDocumentPicks.addRandomField(random(), ingestDocument, originalValue);
 
         List<Object> valuesToAppend = new ArrayList<>();
-        String newValue = randomAlphaOfLengthBetween(1, 10);
+        String newValue = originalValue + randomAlphaOfLengthBetween(1, 10);
```

Comment:
Maybe use `randomValueOtherThan`? It'll communicate intent better, I think

Reply:
Thanks, @probakowski! I didn't know about the `randomValueOtherThan` method.